nc/include/component/PhysicsBody.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "component/Transform.h"
+
+namespace nc
+{
+    struct PhysicsProperties
+    {
+        Vector3 velocity = Vector3::Zero();
+        Vector3 angularVelocity = Vector3::Zero();
+        Vector3 linearFreedom = Vector3::One();
```

Comment:
Please put a comment above each of these to explain what they are, in this header.

Reply:
Good call, fixed